downstream/modules/platform/ref-system-requirements.adoc

```diff
@@ -23,6 +23,8 @@ h| Ansible | version 2.9 required |
 
 h| RAM | 4Gb minimum |
 
+h| Python | 3.8 or later |
+
```

Comment:
```suggestion

h| Python | 3.8 or later |

h| RAM | 4Gb minimum |

```

Reply:
@ianf77 I swapped the RAM and Python lines to make the table render properly in the build.